bin/heroku_postdeploy

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# This file is being executed once after first deployment at heroku.
+set -e
+set -x
+
+# Compile static assets
+source ./bin/post_compile
+
+CELERY_ALWAYS_EAGER="True" ./manage.py sync_projects --no-commit
```

Comment:
Good call. Maybe you can add a short comment, like:
`Run celery task locally so that it's executed during build`